docs/docs/gatsby-magic.md

```diff
@@ -46,16 +46,11 @@ Here is where you can find more information on [Routing in Gatsby](/docs/routing
 
 ## Redux
 
-Redux is a state management library often used together with React apps. Gatsby is also a React application and uses Redux behind-the-scenes to manage the state of your site. You don’t have to do anything with Redux directly in Gatsby for it to work.
+When you implement a Gatsby [Node API](/docs/node-apis/), you are passed a collection of [actions](/docs/actions/) (equivalent to actions bound with [bindActionCreators in Redux](https://redux.js.org/api/bindactioncreators/)). When using the supplied actions, you are manipulating state that is created, and relied upon, by Gatsby itself such as created pages and webpack config.
```

Comment:
I'm having a few troubles understanding this paragraph. Should we start with a small introduction?

> Redux is a state container and is used inside Gatsby to keep our internal state in one place. Redux gives us a predictable flow on how and when state updates are done. We can easily hook into these state transitions to update multiple actions to keep everything consistent.
> 
> When you implement a Gatsby [Node API](/docs/node-apis/), you are passed a collection of [actions](/docs/actions/). When using the supplied actions, you are manipulating state that is created, and relied upon, by Gatsby itself such as created pages and webpack config.

Reply:
Agreed, I've added this in.